tests/query/test_organization_extension.py

```diff
@@ -18,7 +18,7 @@ def test_organization_extension_query_processing_happy_path():
         {
             "conditions": []
         },
-        TableSource("my_table", ColumnSet([]))
+        TableSource("my_table", ColumnSet([]), [])
```

Comment:
This can be reverted now because of the default, right?

Reply:
yeah, I think I forgot about this one